app/test/frontend/handlers/account_test.dart

```diff
@@ -46,6 +47,7 @@ void main() {
         await issueGet(
           '/my-packages',
           headers: {'cookie': cookie},
+          host: activeConfiguration.primaryApiUri!.host,
         ),
```

Comment:
Do we ever use `issueGet` for other `host` values?

If so, can we at-least just make two `issueGet` methods? I'm also okay, leaving this for later -- there's probably details here I'm not fully understanding :D

And it's fine to do things in steps.

Reply:
> Do we ever use issueGet for other host values?

Yes, we have redirect tests where we have different behavior for different hosts, and we test it this way.

+1 on the migration/splitting this method...